### PR TITLE
feat: add `'year'` key to the response of `Event::get()`

### PR DESCRIPTION
**Pull request by mgerhold**

### CODE REVIEW

File: app/Controllers/Event.php [hunk 1]
```diff
@@ -23,6 +23,7 @@ public function get(int|null $year = null) {
         $speakers = $speakerModel->getPublished($event['id']);
 
         return $this->response->setJSON([
+            'year' => $year,
```
Comment (PurpurTentakel97) [inline app/Controllers/Event.php:26]: I would expect the year inside the event.
Why did you put it directly inside the main json?

**Review by PurpurTentakel97 — CHANGES_REQUESTED** [review]

File: app/Controllers/Event.php (hunk 1 above)
Comment (mgerhold) [reply app/Controllers/Event.php:26]: I just thought that this is the simplest way of implementing it. Do you want me to change it? Shouldn't be difficult.

Comment (PurpurTentakel97) [reply app/Controllers/Event.php:26]: The difference in the frontend is the type where its located after parsing.
https://github.com/TechStreamConference/website-frontend/blob/main/src/types/provideTypes.ts
Its ether inside the event or the Year.
But I think inside the Event would be not bad for the backend later on.

Comment (mgerhold) [reply app/Controllers/Event.php]: Changed it. Take a look :)

**Review by PurpurTentakel97 — APPROVED** [review]

**Pull request merged**